did-method-spec.md

```diff
@@ -0,0 +1,112 @@
+# TangleID DID Method Specification
+
+## About
+
+The TangleID DID method specification conforms to the requirements specified in the DID specification currently published by the W3C Credentials Community Group. For more information about DIDs and DID method specifications, please see the [DID Primer](https://github.com/WebOfTrustInfo/rwot5-boston/blob/master/topics-and-advance-readings/did-primer.md) and [DID Spec](https://w3c-ccg.github.io/did-spec/).
```

Comment:
Why did you put "method" here?

Reply:
This specification is describing the [DID method](https://w3c-ccg.github.io/did-spec/#did-methods), so put the "method" here.

The [Sovrin](https://sovrin-foundation.github.io/sovrin/spec/did-method-spec-template.html) and [Ethr DID](https://github.com/uport-project/ethr-did-resolver/blob/develop/doc/did-method-spec.md) Method Specification also put the "method" in their about section.